service.watchedlist/resources/language/German/strings.po

```diff
@@ -0,0 +1,386 @@
+# XBMC Media Center language file
```

Comment:
please adjust to the new language folder layout /resources/language/resource.language.de_de/strings.po